roles/nat64_appliance/templates/nat64-appliance.yaml.j2

```diff
@@ -0,0 +1,46 @@
+---
+- imagename: nat64-appliance.qcow2
+  elements:
+    - centos
+    - vm
+    - epel
+    - growroot
+    - block-device-efi
+    - package-installs
+    - nat64-router
+  no-tmpfs: {{ cifmw_nat64_appliance_no_tmpfs }}
+  environment:
+    DIB_RELEASE: '9-stream'
+    DIB_PYTHON_VERSION: '3'
+    DIB_IMAGE_SIZE: '2'
+    COMPRESS_IMAGE: '1'
```

Comment:
I vaguely recall issues with using /tmp as the build directory in some environments. Trying this might help:

`    TMP_DIR: '/var/tmp'`

Reply:
Thanks Steve, trying it. :crossed_fingers: the molecule job will work.

Reply:
Using '/var/tmp' did not help :shrug: - I added `--dry-run` when running in molecule.

Reply:
Marking this resolved.

The molecule job now uses `--dry-run` when building the image. Issue is CI environment related, we can create a follow up change to remove `--dry-run` and troubleshoot this issue later.